chart/values.yaml

```diff
@@ -154,6 +154,7 @@ agents:
         cpu: "500m"
         # -- Memory requests for core agents
         memory: "32Mi"
+    fast_rebuild_wait_period: "0s"
```

Comment:
Is the default value supposed to be 0s ?

Reply:
The variable should use camelCase naming convention, please refer other names.

Reply:
Yes default is supposed to be zero, in which case core-agent uses internally defined values and ignores this.
Will change to camelCase.

Reply:
It's better to keep the default value same as the one used by core agent internally

Reply:
The 'default' value used by core-agent internally would also be dynamic. And the current logic checks if cli provided value is zero, then decide upon a default value internally. This is stated in documentation.

Reply:
Shouldn't it be simple 1:1 mapping? If not, what's the calculation, how can the value be figured out dynamically?

Reply:
Recalling the control-plane design discussion around this - the wait value will be taking into consideration factors like replica size, number of children etc because we wouldn't want to wait for same amount of time for all the volumes.

If the non-zero cli value is provided - it overrides internal decisions and we'll take cli value into consideration.
If cli value is zero - coreagent will decide the values.

Reply:
This is confusing. Just set it to "" and if it's "" then don't provide the argument and core agent will use the default value.

Reply:
oh if `""` is allowed then that's good because core agent will consider it zero.

```
   #[clap(long, default_value = "0s")]
    pub(crate) faulted_child_wait_period: humantime::Duration,
```

Reply:
We never wanted anyone to have a 'choice' between full and partial rebuild. That wasn't the idea behind this. This is so that system tests can reduce the total runtime of partial rebuild suite.

Reply:
Could be useful for testing as well or if we find a bug in partial rebuild for example